test/Faker/Calculator/EanTest.php

```diff
@@ -7,75 +7,76 @@
 
 final class EanTest extends TestCase
 {
-    public function Ean8checksumProvider()
+    public function Ean8checksumProvider(): array
     {
-        return array(
-            array('1234567', '0'),
-            array('2345678', '5'),
-            array('3456789', '0'),
-        );
+        return [
+            ['1234567', 0],
+            ['2345678', 5],
+            ['3456789', 0],
+        ];
     }
 
-    public function ean8ValidationProvider()
+    public function ean8ValidationProvider(): array
```

Comment:
These are breaking changed which cannot happen in a minor release. If someone extends the function, and has a different return type.

Reply:
Does BC promise apply to the test classes too?

Reply:
Oops. I didn't realise it was a test class. Sorry!

Reply:
Lets not do a BC change at all at this point